src/include/homestore/vchunk.h

```diff
@@ -34,6 +34,6 @@ class VChunk {
     cshared< Chunk > get_internal_chunk() const;
 
 private:
-    cshared< Chunk > internalChunk;
+    shared< Chunk > internalChunk;
```

Comment:
I suggest to use consistent naming convention of the repo. Homestore doesn't use camel case
internalChunk -> m_internal_chunk

Reply:
sure ， will do that